linear_programming/simplex.py

```diff
@@ -20,31 +20,28 @@
 class Tableau:
     """Operate on simplex tableaus
 
-    >>> t = Tableau(np.array([[-1,-1,0,0,-1],[1,3,1,0,4],[3,1,0,1,4.]]), 2)
+    >>> Tableau(np.array([[-1,-1,0,0,-1],[1,3,1,0,4],[3,1,0,1,4.]]), 2, 2)
     Traceback (most recent call last):
     ...
     ValueError: RHS must be > 0
     """
 
-    def __init__(self, tableau: np.ndarray, n_vars: int) -> None:
+    def __init__(self, tableau: np.ndarray, n_vars: int, n_art_vars: int) -> None:
```

Comment:
What does `art` mean in this context and why does the reader have to guess?  Please use real words even it if makes the variable longer.  https://word.tips/words-start-with/art